### PR TITLE
Support stroke path

### PR DESCRIPTION
**Pull request by wrnrlr**
More changes related to #5 

* Renamed `Geometry::shape` to `Geometry::fill`
* Add `Geometry::stroke`
* Added stroke example to shapes

### CODE REVIEW

File: src/geometry/shape.rs [hunk 1]
```diff
@@ -1,36 +1,32 @@
 use lyon::path::Path;
 use lyon::tessellation::*;
 
-// TODO is this always correct?
-fn bounding_radius(path: Path) -> f32 {
+type PositionBuilder = VertexBuffers<crate::Position, u16>;
+
+fn fill_position(vertex: FillVertex)->crate::Position {
```
Comment (kvark) [inline src/geometry/shape.rs]: please run `cargo fmt` on this!

Comment (kvark) [inline src/geometry/shape.rs]: I guess you could also go `impl From<FillVertex> for crate::Position` if you want fancy. I'm fine either way

File: src/geometry/shape.rs [hunk 2]
```diff
@@ -42,4 +38,22 @@ impl super::Geometry {
             radius,
         }
     }
+
+    pub fn stroke(path: &Path, options:&StrokeOptions) -> Self {
+        let mut buffer = PositionBuilder::new();
+        let builder = &mut BuffersBuilder::new(&mut buffer, stroke_position);
+        let mut tessellator = StrokeTessellator::new();
+        {
+            tessellator.tessellate_path(path, options, builder).unwrap();
```
Comment (kvark) [inline src/geometry/shape.rs]: why are we moving this to a separate scope?

File: src/geometry/shape.rs (hunk 1 above)
Comment (wrnrlr) [reply src/geometry/shape.rs]: I could do something like this, but I don't know how I would use this with the rest of the API

```
impl<'a, 'b> From<StrokeVertex<'a, 'b>> for crate::Position {
  fn from(vertex: StrokeVertex<'a, 'b>) -> Self {
    let p = vertex.position();
    crate::Position([p.x, p.y, 0.0])
  }
}

impl<'l> From<FillVertex<'l>> for crate::Position {
  fn from(vertex: FillVertex<'l>) -> Self {
    let p = vertex.position();
    crate::Position([p.x, p.y, 0.0])
  }
}
```

File: src/geometry/shape.rs (hunk 2 above)
Comment (wrnrlr) [reply src/geometry/shape.rs]: Cargo cult programming on my part, it was like this in the example.

**Review by kvark — APPROVED** [review]

**Pull request merged**